ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/SqlLedgerReaderWriter.scala

```diff
@@ -122,9 +124,25 @@ object SqlLedgerReaderWriter {
     for {
       uninitializedDatabase <- Database.owner(jdbcUrl)
       database = uninitializedDatabase.migrate()
+      queries = database.queries
+      ledgerId <- ResourceOwner.forFuture(() =>
+        database.inWriteTransaction("Checking ledger ID at startup") { implicit connection =>
```

Comment:
Probably worth putting the thunk passed to `ResourceOwner.forFuture` in its own method.

Reply:
Pulled this and the other thunk out. 🙂